mount/go.mod

```diff
@@ -4,5 +4,5 @@ go 1.17
 
 require (
 	github.com/moby/sys/mountinfo v0.6.2
-	golang.org/x/sys v0.0.0-20220520151302-bc2c85ada10a
+	golang.org/x/sys v0.11.0
```

Comment:
if we don't need anything newer, I prefer follow Go module's "minimum version selection", and use the oldest tagged version (v0.1.0), and leave it up to users of this module to pick the version they want

Reply:
Done, PTAL @thaJeztah